.gitignore

```diff
@@ -4,7 +4,7 @@
 *.dll
 *.so
 *.dylib
-main
+parser
```

Comment:
The default binary name is still main, right?

If so - then please consider adding a tiny Makefile rule to compile binary with a wanted name.
If we take into account that binary could be moved outside of the project folder - then devfile-parser could be better name. WDYT?

Reply:
If you do a `go build`, `parser` gets built.   But if you do `go buld main.go`,  `main` is produced.  I guess I should add back `main`.    I'd agree that we should create a Makefile.  In this PR, the goal is to stop binaries getting merged inadvertently.